tests/Microsoft.Identity.Web.Test/AuthorityHelpersTests.cs

```diff
@@ -124,6 +124,22 @@ public void BuildAuthority_AadValidOptions_ReturnsValidAadAuthority()
             Assert.Equal(expectedResult, result);
         }
 
+        [Fact]
+        public void BuildAuthority_AadValidOptionsWithDomain_ReturnsValidAadAuthority()
+        {
+            MicrosoftIdentityOptions options = new MicrosoftIdentityOptions
+            {
+                Domain = TestConstants.Domain,
+                Instance = TestConstants.AadInstance
+            };
+            string expectedResult = $"{options.Instance}/{options.Domain}/v2.0";
+
+            string result = AuthorityHelpers.BuildAuthority(options);
+
+            Assert.NotNull(result);
```

Comment:
I think this is not needed, since nullability will be checked in the `.Equal`. (I realize other test have the same pattern, so maybe those should be updated too then.)